pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -254,7 +254,7 @@ var FileAutoComplete = React.createClass({
 
     renderError: function(error) {
         return (
-            <li className="alert alert-warning">
+            <li key='error' className="alert alert-warning">
```

Comment:
Aside from the inconsistent quoting, what does that tag do? It's not a standard HTML tag, and nothing seems to use it?